.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -31,7 +31,7 @@ If applicable, add screenshots to help explain your problem.
  - OS: [e.g. OS X, Windows]
  - Hardware Wallet: [e.g. Trezor Firmware version 1.8.3, Ledger Nano S Firmware version 1.6.0]
  - Browser: [e.g. Chrome Version 79.0.3945.79 (Official Build) (64-bit), Firefox Browser 71.0 (64-bit)]
- - MetaMask Version: [e.g. 5.0.2]
+ - MetaMask Version: [e.g. 5.0.2 - find it in Settings->About]
```

Comment:
I like this! Could we change this line to:  - MetaMask Version: [e.g. 5.0.2 - find it in Settings > About] - also if you could sign the CLA that would be great :)